main/src/main/java/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -1062,16 +1062,16 @@ static Single<Boolean> removeFromWatchlist(@NonNull final Geocache cache) {
 
     @Nullable
     static String requestHtmlPage(@Nullable final String geocode, @Nullable final String guid, final String log) {
-        final Parameters params = new Parameters("decrypt", "y");
         if (StringUtils.isNotBlank(geocode)) {
-            params.put("wp", geocode);
+            return GCLogin.getInstance().getRequestLogged("https://www.geocaching.com/geocache/" + geocode, null);
```

Comment:
You can use here: GCConstants.GC_URL => "https://www.geocaching.com/"

Reply:
There's tons of places where this could be applied - but it's not related to the actual change here, so not going to change it now - also considering PC is already off and this is an urgent fix.